Gruntfile.js

```diff
@@ -77,8 +94,8 @@ module.exports = (grunt) => {
       },
     },
     eslint: {
-      target: SOURCES.concat(['./tests/**/*.ts', './tests/**/*.js']),
-      options: { fix: true },
+      target: SOURCES.concat('./tests'),
+      options: { fix: false },
```

Comment:
Why don't you want the issues to be fixed? Eslint is quite good at it for example with templates on strings.

Reply:
This makes unnecessary changes which I do not plan to introduce. Github and console shows you all the warnings. They can be fixed separately.